docs/book/architecture.rst

```diff
@@ -170,21 +176,22 @@ Creating a product using custom form and redirection method:
                 form: app_user_product # Use this form type!
                 template: AppStoreBundle:Product:create.html.twig # Use a custom template.
                 factory: 
-                    method: createForStore # Use custom factory method to create a product.
+                    method: createForStore # Use a custom factory method to create a product.
                     arguments: [$store] # Pass the store name from the url.
                 redirect:
                     route: app_product_index # Redirect the user to his products.
                     parameters: [$store]
 
-All other methods have the same level of flexibility and are documented in the [SyliusResourceBundle guide].
+All other methods have the same level of flexibility and are documented in the :doc:`Resource Bundle Guide </bundles/SyliusResourceBundle/index>`.
 
-Core and Web Interface
-----------------------
+Core, Admin and Ui
+------------------
 
-Main application is constructed from two main bundles:
+Main application is constructed from three main bundles:
 
 **SyliusCoreBundle**, which is the glue for all other bundles. It is the integration layer of Core component - the heart of Sylius, providing the whole e-commerce framework.
-**SyliusWebBundle**, which contains the default web interface, assets, templates and menu builders.
+**SyliusAdminBundle**, which contains the default administration of the whole system, which is easily extensible.
```

Comment:
`which... which` repetition's never good. Besides is the last part really necessary?